Makefile

```diff
@@ -275,3 +268,9 @@ openshift-image-mirror-mappings:
 	$(CONTAINER_ENGINE) pull registry.ci.openshift.org/ci/promoted-image-governor:latest
 	$(CONTAINER_ENGINE) run --rm -v "$(CURDIR):/release:z" registry.ci.openshift.org/ci/promoted-image-governor:latest --ci-operator-config-path /release/ci-operator/config --release-controller-mirror-config-dir /release/core-services/release-controller/_releases --openshift-mapping-dir /release/core-services/image-mirroring/openshift --openshift-mapping-config /release/core-services/image-mirroring/openshift/_config.yaml
 .PHONY: openshift-image-mirror-mappings
+
+config_updater_vault_secret:
+	@[[ -z $$cluster ]] && echo "ERROR: \$$cluster must be set" && exit 1
+	oc apply --context=$(cluster) --recursive -f clusters/build-clusters/common
+	make ci-secret-bootstrap
```

Comment:
Shouldnt this be `ci-secret-generator`?
the data have not been uploaded to Vault yet.